eng/common/README.md

```diff
@@ -20,7 +20,6 @@ The 'Sync eng/common directory' PRs will be created in the language repositories
 3. The  `azure-sdk-tools - sync - eng-common` pipeline queues test runs for template pipelines in various languages. These help you test your changes in the **Tools PR**.
 4. If there are changes in the **Tools PR** that will affect the release stage you should approve the release test pipelines by clicking the approval gate. The test (template) pipeline will automatically release the next eligible version without needing manual intervention for the versioning. Please approve your test releases as quickly as possible. A race condition may occur due to someone else queueing the pipeline and going all the way to release using your version while yours is still waiting. If this occurs manually rerun the pipeline that failed.
 5.  If you make additional changes to your **Tools PR** repeat steps 1 - 4 until you have completed the necessary testing of your changes. This includes full releases of the template package, if necessary.
-6. Sign off on CreateSyncPRs stage of the sync pipeline using the approval gate. This stage will create the **Sync PRs** in the various language repos with the `auto-merge` label applied. A link to each of the **Sync PRs** will show up in the **Tools PR** for you to click and review.
-7. Go review and approve each of your **Sync PRs**. The merging will happen automatically.
-8. Sign off on VerifyAndMerge stage of the sync pipeline using the approval gate. This stage will merge any remaining open **Sync PRs** and also append `auto-merge` to the **Tools PR** so it will automatically merge once the pipeline finishes.
-7. Sign Off on the VerifyAndMerge stage. This will merge any remaining open **Sync PR** and also append `auto-merge` to the **Tools PR**.
+6. Sign off on CreateSyncPRs stage of the sync pipeline using the approval gate. This stage will create the **Sync PRs** in the various language repos. A link to each of the **Sync PRs** will show up in the **Tools PR** for you to click and review.
```

Comment:
Currently it seems like we need to also do a release of the template in order to get the Sync PR's green in the languages we use the template pipelines. Is this something you are still trying to figure out? If it is something that is required we should add that information in these docs. 

cc @benbp

Reply:
I found a few times today where I was going through the motions of doing a release even though I wasn't actually touching anything that impacted the release pipeline directly. I think we should document the break glass procedure for these scenarios (e.g. merging the PRs even without the pipelines completing). It requires admin rights which means it will get some scrutiny, except when we do it :|

Reply:
The other break glass option is to override check-enforcer. However I think I would like to see if we can figure out a way to not have this block.

Reply:
It should be fixed by these PRs.
https://github.com/Azure/azure-sdk-for-net/pull/16145
https://github.com/Azure/azure-sdk-for-js/pull/11986
https://github.com/Azure/azure-sdk-for-java/pull/16657
https://github.com/Azure/azure-sdk-for-python/pull/14686

Reply:
Do we have any end-to-end tests that verify we no longer block the sync PRs?

Reply:
Am testing now with another change.

Reply:
I assume your tests came back positive? So the workflow no longer blocks if a release isn't approved?

Reply:
We override the yaml configuration on the pipeline. So I added a manual branch exclusion to the pipeline UI. In my test it still added the internal template run as part of the PR checks, but it showed up green and did not block the pipeline. I still need more example tp know if that will be its consistent behavior.

Reply:
I created a separate issue to track what I believe is being discussed here:

#1147 Sync PRs should be auto-mergable if check-enforcer is green